checkbox-support/checkbox_support/scripts/image_checker.py

```diff
@@ -42,12 +66,15 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-t", "--type", action="store_true")
     parser.add_argument("-s", "--source", action="store_true")
+    parser.add_argument("-d", "--detect_desktop", action="store_true")
     args = parser.parse_args()
 
     if args.type:
         print("type: {}".format(get_type()))
     if args.source:
         print("source: {}".format(get_source()))
+    if args.detect_desktop:
+        print("Has desktop environment? {}".format(has_desktop_environment()))
```

Comment:
The previous lines seem to be a resource-like output, should this be as well?

Reply:
By "resource-like" did you mean replacing L77 with:
```
has_desktop: true
```
so that it looks like a key-value pair, or did you mean changing the return value of `has_desktop_environment` to a string to match the other functions?

Reply:
first option, it seems to me that the output is a (key value pair) that can be used (I don't know if it actually is used) to build a resource. With the output you added, that purpose would no longer work, and it would also be pretty strange to have such a discrepancy.

Reply:
Makes sense, I can add a comment as well to let people know that we should print key-value pairs :+1: